config-x86-gpu.ini

```diff
@@ -61,8 +61,8 @@ DailyReportTime = 06:00
 [Detector]
 ; Supported devices: Jetson , EdgeTPU, Dummy, x86, x86-gpu
 Device = x86-gpu
-; Supported models: mobilenet_ssd_v2 and openpifpaf
-Name = openpifpaf
+; Supported models: mobilenet_ssd_v2 , openpifpaf and alphapose_mobilenet_ssd
+Name = alphapose_mobilenet_ssd
 ;ImageSize should be 3 numbers seperated by commas, no spaces: 300,300,3 (for better accuracy use higher resolution when
 ; using openpifpaf (openpifpaf detects both faces and pedestrians)
 ImageSize = 1281,721,3
```

Comment:
The ImageSize is used for Alphapose?
I think leave a comment for ImageSize and its usage can be helpful.